new-integration-tests/src/test/java/oracle/weblogic/kubernetes/annotations/Namespaces.java

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package oracle.weblogic.kubernetes.annotations;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation interface to define the number of namespaces needed by the test.
+ */
+@Target({ElementType.PARAMETER})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Namespaces {
+  int value() default 0;
```

Comment:
Why default to 0 instead of 1?  Wouldn't zero be when the annotation is not used.

Reply:
I wanted the test developers to get used to mentioning the number of namespaces they require. If no values I don't know what this annotation means.( at least for me)